http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcJsonConfiguration.java

```diff
@@ -38,15 +38,18 @@
         "resource", "public-client", "credentials",
         "use-resource-role-mappings", "use-realm-role-mappings",
         "enable-cors", "cors-max-age", "cors-allowed-methods", "cors-exposed-headers",
-        "expose-token", "bearer-only", "autodetect-bearer-only",
-        "connection-pool-size",
+        "expose-token", "bearer-only", "autodetect-bearer-only", "connection-pool-size",
```

Comment:
Not necessarily as part of this PR but we seem to have String constants being repeated - should these reference common constants defined somewhere?

Reply:
I believe some (if not all) are constants inside Oidc.java class. So, we might be able to use those instead.